examples/README.md

```diff
@@ -11,25 +11,43 @@ you can install all of the dependencies in the requirements.txt:
 ```bash
 pip install -r requirements.txt
 ```
+### Decide
+
+The `/decide` endpoint returns the requested flag decisions for a user. The flag decision includes flag enabled/disabled status and flag variation.
+
+Example usage:
+```bash
+python basic.py <SDK-Key>
+```
+
+The `/decide` endpoint also supports batching requests by supplying multiple flag keys in 
+single `/decide` call. Please refer to [advanced.py](./advanced.py) for an example on how to fetch multiple decisions. This
+is useful to reduce the number of outbound API calls made from your service to Optimizely Agent.
+
+Example usage:
+```bash
+python advanced.py <SDK-Key>
+```
 
 ### Activate
 
 The `/activate` endpoint returns a decision of the requested experiment or feature for a given user context.
-For single decisions please refer to [basic.py](./basic.py) which demonstrates how to iterate through the configuration for
+For single decisions please refer to [basic_activate.py](./basic_activate.py) which demonstrates how to iterate through the configuration for
 a given SDK project and make individual activation requests.
+Activate is currently still widely used, but it will be gradually superseeded by Decide.
```

Comment:
What about making this deprecation note more clearly as a heading of "Activate" and move it to the bottom of the page. It will be less confusing about similar APIs.
For all other SDKs, we replace the old APIs with decide API samples. For consistency, that should be considered too.

Reply:
@jaeopt I updated per your request. pls review again

Reply:
Actually pls wait with re-review, still need to address your other comments.